mm2src/coins/nft.rs

```diff
@@ -413,6 +595,17 @@ async fn send_request_to_uri(uri: &str) -> MmResult<Json, GetInfoFromUriError> {
     Ok(body)
 }
 
+async fn send_post_request_to_uri(uri: &str, body: String) -> MmResult<Vec<u8>, GetInfoFromUriError> {
+    let (status, _header, body) = slurp_post_json(uri, body).await?;
+    if !status.is_success() {
+        return Err(MmError::new(GetInfoFromUriError::Transport(format!(
+            "Response !200 from {}: {}",
+            uri, status,
+        ))));
+    }
+    Ok(body)
+}
```

Comment:
I have some notes on this:

- This and other request related functions in this module seems very useful for general purpose and should be under net module with other request functions.
- Log should be updated to `2**`, whole range of 200-299 is success status code for HTTP.
- We should state that function will return error if request status code is not `2**`. Maybe in the function name if possible, or with the doc-comments.

Reply:
moved `send_post_request_to_uri` to `transport` and `send_request_to_uri` to `wasm_http` and `native_http`. decided to mention 2xx range in doc comment.